Clappr.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
   s.name             = "Clappr"
   s.version          = "0.7.3"
-  s.summary          = "An extensible media player for iOS"
+  s.summary          = "An extensible media player for iOS and tvOS"
```

Comment:
it seems that the previous Clappr for tvOS version was 0.8.1 as you can check in the related branch podspec https://github.com/clappr/clappr-ios/blob/tvos/Clappr.podspec. Maybe we can use 0.9 for the multi-platform podspec version

Reply:
Hey guys, shouldn't we update only when we are in a release process?
But I totally agree that it can become `0.9` when we release the new version 😄